src/index.ts

```diff
@@ -30,6 +31,11 @@ const createNewProject = createNewProjectCommandFactory({
   updateStoreConfig,
 });
 
+const runCommand = <T>(command: Command<T>, options: T): Promise<void> =>
+  Promise.resolve(command(options)).catch((error: Error) =>
```

Comment:
that's quite an interesting approach, at first glance I was surprised you removed try/catch blocks, but that makes sense :)